docs/components/connectors/connector-types.md

```diff
@@ -14,13 +14,28 @@ Looking for pre-built, [Out-of-the-box Connectors](/components/connectors/out-of
 
 Outbound Connectors allow workflows to trigger external systems or services, making it possible to integrate workflows with other parts of a business process or system architecture.
 
+The programming code in Java to connect to the external system is executed when the workflow reaches the service task.
+
 ![Outbound Connectors](img/outbound-connectors.png)
 
 Use outbound Connectors if something needs to happen in the third-party system if a process reaches a service task. For example, calling a REST endpoint or publishing a message to Slack.
 
 ## Inbound Connectors
 
 Inbound Connectors enable workflows to receive data or messages from external systems or services, making it possible to integrate workflows into a wider business process or system architecture.
+Inbound Connectors can be used to trigger a process instance, or to send a message to a running process instance.
+
+The Java code of the inbound Connector has a lifecycle suitable for long-running operations, such as listening for messages on a queue or waiting for a webhook to be called.
```

Comment:
```suggestion
The Java code of the inbound Connector get `activated` as soon as the Connector Runtime detects an element in a process definition that references an inbound connector. The lifecycle suitable for long-running operations, such as listening for messages on a queue or waiting for a webhook to be called. The code gets `deactivated` in case of an updated or deleted process definition.
```